src/views/RecIM/components/List/Listing/Listing.module.scss

```diff
@@ -2,18 +2,51 @@
 @import '../../../../../vars';
 
 .listing {
-  background-color: $neutral-light-gray2-opacity10;
+  @extend %baseListing;
   margin-top: 0.1em;
   margin-bottom: 0.1em;
   padding: 1em;
+}
+
+//trial new listing look
+.listing:hover {
+  background-color: $neutral-light-gray2-opacity40;
+  box-shadow: inset 1px 1px $neutral-black-opacity10;
+  transition: all 0.3s ease-in-out;
+}
+
+%baseListing {
+  background-color: $neutral-light-gray2-opacity10;
```

Comment:
A couple of the colors here in listing:hover and baseListing appear to have been added by you, and should probably be updated to the new variables.

Reply:
That was in another PR, I'll adjust them on the appropriate PR and merge those changes in here.